internal/exec/resolvable/resolvable.go

```diff
@@ -207,40 +216,67 @@ func (b *execBuilder) makeObjectExec(typeName string, fields schema.FieldList, p
 	methodHasReceiver := resolverType.Kind() != reflect.Interface
 
 	Fields := make(map[string]*Field)
+	rt := unwrapPtr(resolverType)
 	for _, f := range fields {
-		methodIndex := findMethod(resolverType, f.Name)
-		if methodIndex == -1 {
+		methodIndex := -1
+		fieldIndex := -1
+
+		// 1) Use resolver type's method when
+		//	1.1) __Type and __Schema requests
+		// 	1.2) Or field has arguments
+		//	1.3) Or it is configured to use method
+		//	1.4) Or it is an interface
+		// 2) Otherwise use resolver type's field
+		if isResolverSchemaOrType(rt) == true || len(f.Args) > 0 ||
+			b.schema.UseFieldResolvers == false || rt.Kind() == reflect.Interface {
```

Comment:
`UseFieldResolver` is already a boolean, right? Can't you just use `!b.schema.UseFieldResolvers`? Or you want it to be more verbose?

Reply:
imho, `!` can be overlooked sometime so more verbose make it clear that it's a check for `false`. I can change this to `!`

Reply:
I prefer using `!`, but it's up to the owners of the library to decide.

Reply:
I prefer using `!` because it'll be consistent with the rest of the code in this library.